qa/vagrant-helpers.rb

```diff
@@ -1,15 +1,26 @@
 # encoding: utf-8
 require "open3"
+require "json"
 
 module LogStash
   class VagrantHelpers
+    DEFAULT_CENTOS_BOXES = [ "centos-6", "centos-7" ]
+    DEFAULT_DEBIAN_BOXES = [ "ubuntu-1204", "ubuntu-1404"]
 
-    def self.bootstrap
-      execute("vagrant up")
+    def self.bootstrap(box)
+      Dir.chdir("acceptance") do
+        p execute("vagrant up #{box} --provider virtualbox")
+        raw_ssh_config    = execute("vagrant ssh-config")[:stdout].split("\n");
+        parsed_ssh_config = parse(raw_ssh_config)
+        File.write(".vm_ssh_config", parsed_ssh_config.to_json)
```

Comment:
Well, this is a bit of story here, let me explain my thoughts .... When I first thought about it my idea is/was to decouple everything as much as possible, so dumping the necessary ssh connection is something kinda necessary to do here.

On the other side, the only thing I can see is polling this same call for each time we run an acceptance test, and this would somehow slowdown the process. What do you think?